test/format/format.sh

```diff
@@ -195,11 +197,12 @@ config=$(find_file "$config")
 	exit 1
 }
 
-verbose "$name configuration: [-c $config] [-h $home]\
-[-j $parallel_jobs] [-n $total_jobs] [-t $minutes] $format_args"
+verbose "$name configuration: $format_binary [-c $config]\
+[-h $home] [-j $parallel_jobs] [-n $total_jobs] [-t $minutes] $format_args"
```

Comment:
Is the purpose of introducing `--` new command argument to separate format arguments from other arguments? And a `--` is suggested to be put in front of `$format_args`  for clarity (though it's optional)?

Reply:
The only purpose is to separate `format.sh` arguments from `format` arguments, so, for example, if you want to pass `-l` to `format`, you have to be sure `format.sh` doesn't parse it, so you use `--`.